basic UI setup for dark theme

paul4702 opened the pull request:
Hi team, 

If you want to activate the dark mode  

got to 

src\lib\components\header\sub-components\SidebarLinksGroup.svelte , line 276

and uncomment the switch to switch between dark and light theme.

I have cross checked with the current UI of dev and my changes. So that there  shouldn't be any diff. 

If you find any please leave a comment will get on it immediately.

souvikmishra commented on src/lib/page-components/oyster/dashboard/OysterDashboardPage.svelte:
```diff
@@ -99,8 +99,8 @@
-	<h2 class="text-3xl font-medium leading-[-2px] text-[#030115]">
-		Hello, {$connected ? shortenText($walletStore.address, 6, 6) : 'Fishy'}
+	<h2 class="text-3xl font-medium leading-[-2px] text-grey-800">
+		Hello, {shortenText($walletStore.address, 6, 6)}
```
is there a reason why we removed the `$connected` check?

paul4702 replied:
![image](https://github.com/marlinprotocol/interface-v2/assets/162085914/34ed0906-8731-45af-82eb-9201eacb0239)


I have no idea, How this line was missed. I am updating it

souvikmishra commented on src/lib/components/inputs/AmountInputWithMaxButton.svelte:
```diff
@@ -23,9 +23,8 @@
-				disabled={!handleUpdatedAmount || !$connected || disabled}
-				{onlyInteger}
+				styleClass="input input-ghost input-primary text-3xl placeholder:text-grey-800/[.2]"
+				disabled={!handleUpdatedAmount || !$connected}
```
is there a reason why we removed `|| disabled` and `{onlyInteger}`

souvikmishra commented:
also, it feels like some of the classes that we have replaced the hardcoded with are not in both light and dark configs, maybe check once

paul4702 commented:
> also, it seems like some of the classes that we have replaced the hardcoded colors with are not present in both light and dark configs, maybe check once @paul4702

@souvikmishra  The colors we have changed you can find them under the variable names ex: for grey-800 :  --grey-800 & you can find --grey-800 assigened to grey-800 under theme extended colors: PFA screenshot 

 I have changed the most used 6 colors 


![image](https://github.com/marlinprotocol/interface-v2/assets/162085914/bada1856-5776-4f2b-9a6b-868d81f96c18)

![image](https://github.com/marlinprotocol/interface-v2/assets/162085914/2752dd9a-420d-4050-b142-3fef35b5ad20)


![image](https://github.com/marlinprotocol/interface-v2/assets/162085914/52bf074b-19b3-4857-8080-ec5c57ef119c)

vishwajeetraj11 commented on src/lib/components/header/sub-components/SidebarLinksGroup.svelte:
```diff
@@ -134,11 +134,13 @@
 		}
 	];
+
+	$: isDarkModeActive = localStorage?.getItem('theme') === 'light' ? false : true;
```
this check can be improved.
```$: isLightModeActive = localStorage?.getItem('theme') === 'light'```
or (use dark mode constant)
```$: isDarkModeActive = localStorage?.getItem('theme') === 'dark'```

paul4702 replied:
updated the condition

paul4702 commented on src/lib/components/header/sub-components/SidebarLinksGroup.svelte:
```diff
@@ -271,6 +273,28 @@
 			{/each}
 		</ul>
+		<!----------------------------------------------------------------------------------->
+		<!-- Uncomment the following code to enable theme switcher -->
```
yes, This is a switch so, we want the code to be merged in steps and if somebody wants to test the dark theme they can uncomment it and check how it is looking as it we are pushing it to dev

paul4702 commented on src/lib/page-components/dashboard/MarlinHubDashboard.svelte:
```diff
@@ -114,10 +114,10 @@
-					<div class="flex items-center gap-2 text-2xl font-medium text-[#030115]">
-						<img src={card.logo} alt={card.logoAlt} />{card.title}
+					<div class="flex items-center gap-2 text-2xl font-medium text-grey-800">
+						<img src={card.logo} alt={card.logoAlt} class="dark:bg-white" />{card.title}
```
Yes, It was but I am trying out a diff approach to make sure the icons look invert in color. removed it and updated the code
